tests/unit/test_webhook_model.py

```diff
@@ -76,7 +76,7 @@ def test_model_instance():
 @patch('webcompat.webhooks.model.make_request')
 def test_close_private_issue(mock_mr):
     """Test issue state and API request that is sent to GitHub."""
-    mock_mr.return_value.status_code == 200
+    mock_mr.return_value.status_code = 200
```

Comment:
So good to have caught this. Now I'm worried… because it doesn't change anything. I wonder what was the value then and how it would modify the test or not. Let's see.

Old:

```
tests/unit/test_webhook_model.py::test_close_private_issue <MagicMock name='make_request().status_code' id='4363956624'>
PASSED
```

New:

```
tests/unit/test_webhook_model.py::test_close_private_issue 200
PASSED
```

ok definitely wrong.

After removing them, everything is still passing.
So we don't test the status code in any ways. 

I think we can safely remove all instances of `mock_mr.return_value.status_code = 200`

Reply:
Yeah I was thinking that too :) I'll remove them